uplink/ecclient/client.go

```diff
@@ -263,6 +267,11 @@ func (ec *ecClient) Repair(ctx context.Context, limits []*pb.AddressedOrderLimit
 
 	ec.log.Sugar().Infof("Successfully repaired %s to %d nodes.", path, atomic.LoadInt32(&successfulCount))
 
+	mon.IntVal("repair_segment_pieces_total").Observe(int64(pieceCount))
```

Comment:
there might be some overlap between these stats and the stats sent from `satellite/repair/repairer/segments.go`. It might be worth comparing

Reply:
These were all the things that I saw in there:
```
mon.Meter("repair_attempts").Mark(1)
...
mon.IntVal("repair_segment_size").Observe(pointer.GetSegmentSize())
...
if int32(numHealthy) < pointer.Remote.Redundancy.MinReq+1 {
    mon.Meter("repair_nodes_unavailable").Mark(1)
    ...
}
...
if int32(numHealthy) > pointer.Remote.Redundancy.RepairThreshold {
    mon.Meter("repair_unnecessary").Mark(1)
    ...
}
...
mon.FloatVal("healthy_ratio_before_repair").Observe(healthyRatioBeforeRepair)
...
healthyAfterRepair := int32(len(healthyPieces) + len(repairedPieces))
switch {
case healthyAfterRepair <= pointer.Remote.Redundancy.RepairThreshold:
    mon.Meter("repair_failed").Mark(1)
case healthyAfterRepair < pointer.Remote.Redundancy.SuccessThreshold:
    mon.Meter("repair_partial").Mark(1)
default:
    mon.Meter("repair_success").Mark(1)
}
...
mon.FloatVal("healthy_ratio_after_repair").Observe(healthyRatioAfterRepair)
```

Unless I'm misunderstanding monkit again, it looked to me like the segment.go stuff was mostly `mon.Meter(...).Mark(1)` and none of the observers overlapped exactly so I went for it.

Reply:
ok yeah you're right, there doesn't seem to be overlap